### PR TITLE
chore(linters): Fix findings found by testifylint: bool-compare

### PR DESCRIPTION
**Pull request by zak-pawel**
Address findings for [testifylint: bool-compare](https://github.com/Antonboom/testifylint#bool-compare) - checks usage of github.com/stretchr/testify.

```
❌   assert.Equal(t, true, result)
     assert.NotEqual(t, true, result)
     assert.False(t, !result)
     assert.True(t, result == true)
     // And other variations...

✅   assert.True(t, result)
     assert.False(t, result)
```

It is only part of the bigger job.
After all type of findings in whole project are handled, we can enable `testifylint` linter in `golangci-lint`.

### CODE REVIEW

File: plugins/inputs/aerospike/aerospike_test.go [hunk 1]
```diff
@@ -452,7 +452,12 @@ func TestAerospikeParseValue(t *testing.T) {
 	require.Equal(t, uint64(18446744041841121751), val)
 
 	val = parseAerospikeValue("", "true")
-	require.Equal(t, true, val)
+	switch v := val.(type) {
+	case bool:
+		require.True(t, v)
+	default:
+		require.Fail(t, fmt.Sprintf("bool type expected, got '%T' with '%v' value instead", val, val))
+	}
```
Comment (srebhan) [inline plugins/inputs/aerospike/aerospike_test.go:460]: How about
```suggestion
	v, ok := val.(bool)
	require.Truef(t, ok, "bool type expected, got '%T' with '%v' value instead", val, val)
	require.True(t, v)
```

File: plugins/inputs/couchbase/couchbase_test.go [hunk 2]
```diff
@@ -210,7 +211,12 @@ func TestGatherFailover(t *testing.T) {
 
 	require.NotNil(t, metric)
 	require.Equal(t, 1, metric.Fields["count"])
-	require.Equal(t, true, metric.Fields["enabled"])
+	switch v := metric.Fields["enabled"].(type) {
+	case bool:
+		require.True(t, v)
+	default:
+		require.Fail(t, fmt.Sprintf("bool type expected, got '%T' with '%v' value instead", metric.Fields["enabled"], metric.Fields["enabled"]))
+	}
```
Comment (srebhan) [inline plugins/inputs/couchbase/couchbase_test.go:219]: See comment for `aerospike`.

File: plugins/inputs/github/github_test.go [hunk 3]
```diff
@@ -103,7 +103,7 @@ func TestGetTags(t *testing.T) {
 		"license":  licenseName,
 	}
 
-	require.Equal(t, true, reflect.DeepEqual(getTagsReturn, correctTagsReturn))
+	require.True(t, reflect.DeepEqual(getTagsReturn, correctTagsReturn))
```
Comment (srebhan) [inline plugins/inputs/github/github_test.go:106]: Hmmm shouldn't
```suggestion
	require.Equal(t, correctTagsReturn, getTagsReturn)
```
this be sufficient? IIRC `require.Equal` _does_ a deep equal...

File: plugins/inputs/github/github_test.go [hunk 4]
```diff
@@ -136,5 +136,5 @@ func TestGetFields(t *testing.T) {
 	correctFieldReturn["subscribers"] = 5
 	correctFieldReturn["watchers"] = 6
 
-	require.Equal(t, true, reflect.DeepEqual(getFieldsReturn, correctFieldReturn))
+	require.True(t, reflect.DeepEqual(getFieldsReturn, correctFieldReturn))
```
Comment (srebhan) [inline plugins/inputs/github/github_test.go:139]: See above.

File: plugins/inputs/knx_listener/knx_listener_test.go [hunk 5]
```diff
@@ -179,10 +179,20 @@ func TestRegularReceives_MultipleMessages(t *testing.T) {
 	require.Equal(t, "temperature", acc.Metrics[0].Measurement)
 	require.Equal(t, "1/1/1", acc.Metrics[0].Tags["groupaddress"])
 	require.Len(t, acc.Metrics[0].Fields, 1)
-	require.Equal(t, true, acc.Metrics[0].Fields["value"])
+	switch v := acc.Metrics[0].Fields["value"].(type) {
+	case bool:
+		require.True(t, v)
+	default:
+		require.Fail(t, fmt.Sprintf("bool type expected, got '%T' with '%v' value instead", acc.Metrics[0].Fields["value"], acc.Metrics[0].Fields["value"]))
+	}
```
Comment (srebhan) [inline plugins/inputs/knx_listener/knx_listener_test.go:187]: See comment for `aerospike`.

**Review by srebhan — COMMENTED** [review]
Thanks for the epic work @zak-pawel! Just some suggestions from my side...

File: plugins/inputs/aerospike/aerospike_test.go (hunk 1 above)
Comment (zak-pawel) [reply plugins/inputs/aerospike/aerospike_test.go]: Nice, changed!

File: plugins/inputs/github/github_test.go (hunk 3 above)
Comment (zak-pawel) [reply plugins/inputs/github/github_test.go]: Changed.

File: plugins/inputs/github/github_test.go (hunk 4 above)
Comment (zak-pawel) [reply plugins/inputs/github/github_test.go]: Changed.

File: plugins/inputs/knx_listener/knx_listener_test.go (hunk 5 above)
Comment (zak-pawel) [reply plugins/inputs/knx_listener/knx_listener_test.go]: Changed.

**Review by powersj — APPROVED** [review]

**Pull request merged**